woocommerce/woocommerce-android
# Add Order creation view model unit test coverage

Summary
==========
Fixes issue #6047 by creating a full unit test coverage for all main scenarios of the `OrderCreationViewModel`

How to Test
==========
1. Simply try to modify the internal business logic of the `OrderCreationViewModel` and verify if the tests catch the modification by failing.

Update release notes:
